test/src/eks_blueprints_e2e_test.go

```diff
@@ -0,0 +1,325 @@
+//go:build e2e
+// +build e2e
+
+package src
+
+import (
+	"context"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/gruntwork-io/terratest/modules/terraform"
+	test_structure "github.com/gruntwork-io/terratest/modules/test-structure"
+	"github.com/stretchr/testify/assert"
+	core "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"strings"
+	"testing"
+	internal "github.com/aws-ia/terraform-aws-eks-blueprints/aws"
+)
+
+var (
+	//Test Driven tests Inputs https://github.com/golang/go/wiki/TableDrivenTests
+	testCases = []struct {
+		name   string
+		region string
+		eks_cluster string
+		values map[string]string
+	}{
+		{
+			"eks-cluster-with-new-vpc",
+			"us-west-2",
+			"aws-terra-test-eks",
+			map[string]string{
+				"rootFolder": "../..",
+				"exampleFolderPath": "examples/eks-cluster-with-new-vpc"},
+		},
+	}
+	/* Commented for future use
+	S3BackendConfig = map[string]string{
+		"bucketName": "terraform-ssp-github-actions-state",
+		"s3Prefix": "terratest/examples/",
+		"awsRegion" : "us-west-2"}*/
+
+	destroyModules = []string{
+		"module.kubernetes_addons",
+		"module.eks_blueprints",
+		"module.aws_vpc",
+		"full_destroy"}
```

Comment:
closing bracket on new line

Reply:
Thanks for spotting this. `Go fmt`  magically fixed the issue